bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -225,8 +225,7 @@ void ReplicaImp::tryToSendPrePrepareMsg(bool batchingLogic) {
 
   if (primaryLastUsedSeqNum + 1 > lastStableSeqNum + kWorkWindowSize) return;
 
-  if (primaryLastUsedSeqNum + 1 > lastExecutedSeqNum + config_.concurrencyLevel)
```

Comment:
The changes in this file only seem to be formatting related. Please revert if they don't introduce actual optimizations.